### DISCUSSION TITLE
VSscript to edit 100's of docx document

### DISCUSSION THREAD

**Discussion by amontelo** [microsoft/vscode-discussions#169] [Extension Development QnA] [answered]
Hello all,
I've never used .vb before. Anyways, I have to edit 100's of existing docx file to create a new version of the existing docx. The docx file is a document enlisting how a customer part was build; I want to edit 100's of files with some changes to go on the "new" customer part's name and date of new creation. For that I chose to use .vb script to open, find and substitute the new strings in and save as new file. 
I copied this script to test it and build from there; script opens docx, replace a single string and save it. However, I get this error below once it is launched. I have reviewed the vcb.exec issues and my antivirus log isn't showing this file being the cause, any ideas how to resolve this issue?

.vb script below:
 Dim lOpenFile As Long
 Dim sFileText As String
 Dim sFileName As String

sFileName = "C:\Users\PSLLOCAL\Documents\MOR_TEST\MOR_########_####_##.docx"

 'open the file and read it into a variable
 lOpenFile = FreeFile
 Open sFileName For Input As lOpenFile
 sFileText = Input(LOF(lOpenFile), lOpenFile)
 Close lOpenFile

'change 'Don Trionett' to 'TOPPAN'
 sFileText = Replace(sFileText, " Don Trionett ", " TOPPAN ")

 'write it back to the file
 lOpenFile = FreeFile
 Open sFileName For Output As lOpenFile
 Print #lOpenFile, sFileText
 Close lOpenFile

Error message:
[Running] cd "c:\Users\PSLLOCAL\Documents\MOR_TEST\" && vbc /nologo tempCodeRunnerFile.vb && "c:\Users\PSLLOCAL\Documents\MOR_TEST\"tempCodeRunnerFile
'vbc' is not recognized as an internal or external command,
operable program or batch file.

[Done] exited with code=1 in 0.061 seconds

**Comment by digitarald** [comment, marked as answer]
Please use https://stackoverflow.com/questions/tagged/visual-studio-code to ask end-user related questions.
